src/point.py

```diff
@@ -0,0 +1,23 @@
+import math
+
+class Point:
+    def __init__(self, x: int, y: int) -> None:
+        self.x: int = x
+        self.y: int = y
+
+    # xxxXXXxxx WAAAAAAAAAARRRRRNINGGGGGGGG xxxXXXxxx
+    # Position is passed to properties as string,
+    # so frontends MUST parse the string on their own.
+    # It's not another JSON dictionary for reasons.
+    # That is - C# casting magic :)
+    def __str__(self) -> str:
+        return f"({self.x},{self.y})"
+
+    def __add__(self, other):
```

Comment:
Missing type hinting.

Reply:
I think you cannot specify type `Point` inside `Point` class definition, but I might be wrong.